tests/generators/generators.exp

```diff
@@ -9,10 +9,6 @@ class.js:15:8,8: number
 This type is incompatible with
 class.js:15:12,17: string
 
-class.js:23:39,44: number
```

Comment:
This is a result of the variance change. If you look elsewhere in this file, you'll see the same error in reverse. We no longer unify, so we only get the error "one way." I called this out in the individual commit message, which I generally pack with insights, so don't miss out!